### PR TITLE
Update Section 1 Logic, Section 19 Features Added, Handler Fixes, Prelim Fixes

### PR DESCRIPTION
**Pull request by MrSteve81**
**Overall Review of Changes:**
Update Section 1 Logic
Fixed prelim Tags
Updated Handlers

**Issue Fixes:**
Please list (using linking) any open issues this PR addresses

**Enhancements:**
Added Reboot Option And Section 19 Enhancement For Hives

**How has this been tested?:**
Please give an overview of how these changes were tested. If they were not please use N/A

### CODE REVIEW

File: tasks/prelim.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@
 # Current list is elastic and will be updated as we test more cloud based services.
 # Current testing is working in Azure using Hyper-V. We are currently using this for reference:
 # https://github.com/ansible/ansible/blob/905131fc76a07cf89dbc8d33e7a4910da3f10a16/lib/ansible/module_utils/facts/virtual/linux.py#L205
-- name: Set Fact If Cloud-Based System.
+- name: PRELIM | Set Fact If Cloud-Based System.
```
Comment (mfortin) [inline tasks/prelim.yml:29]: This should only be done on Azure VMs. Amazon EC2 instances should stick with the default Section 01 order.

**Review by mfortin — COMMENTED** [review]
https://github.com/ansible-lockdown/Windows-2019-CIS/pull/98/files#diff-23e21b6c89468f7534697ad091835b3ee5e0213b37ace489488234e5a9548ec4R765-R773
18.6.4.1 is defined twice and the one about DoH should be removed as there is no such control in Windows Server 2019

File: tasks/prelim.yml [hunk 2]
```diff
@@ -36,16 +36,52 @@
...
       - always
 
-- name: Set Windows installation type
+- name: PRELIM | Set Windows installation type
   ansible.builtin.set_fact:
       windows_installation_type: "{{ get_windows_installation_type.value | default('') }}"
   tags:
       - always
+
+- name: PRELIM | Obtatin And Load Default Hive As Well As User Hives
```
Comment (mfortin) [inline tasks/prelim.yml:53]:
```suggestion
- name: PRELIM | Obtain And Load Default Hive As Well As User Hives
```

Comment (frederickw082922) [reply tasks/prelim.yml:53]: Thank you @mfortin ! Nice catch!

**Pull request closed without merging**